### PR TITLE
Add the deprecation notice in all AR v1 docs

### PR DESCRIPTION
**Pull request by FaithOmbongi**

### CODE REVIEW

File: api-reference/beta/api/approval-get.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
+[!INCLUDE [accessreviews-disclaimer](../../includes/accessreviews-disclaimer.md)]
```
Comment (isabelleatmsft) [inline api-reference/beta/api/approval-get.md:16]:
```suggestion
```

Comment (isabelleatmsft) [reply api-reference/beta/api/approval-get.md:16]: @FaithOmbongi this looks like an ELM doc, not access reviews. LMK if you disagree

Comment (isabelleatmsft) [reply api-reference/beta/api/approval-get.md:16]: @FaithOmbongi Looks like this is ELM docs, not AR. LMK if you disagree

**Review by isabelleatmsft — CHANGES_REQUESTED** [review]

**Comment by Lauragra** [comment]
@FaithOmbongi , it looks like Isabelle left some comments after you set Ready to merge. Please verify that we're ready to merge this PR? Thanks.

File: api-reference/beta/api/approval-get.md (hunk 1 above)
Comment (FaithOmbongi) [reply api-reference/beta/api/approval-get.md:16]: Good catch, Isabelle. Yes, you're right. Pushing a commit to resolve this.

**Comment by FaithOmbongi** [comment]
Hi, @Lauragra—Isabelle's feedback was right. I've addressed it.

**Review by isabelleatmsft — APPROVED** [review]

**Pull request merged**